apis/r/tests/testthat/test-SOMADataFrame.R

```diff
@@ -411,25 +417,30 @@ test_that("platform_config is respected", {
   sdf <- SOMADataFrameCreate(uri=uri, schema=asch, index_column_names=c("soma_joinid"), platform_config = cfg)
 
   # Read back and check the array schema against the tiledb create options
+  # XXX CHANGEME
   arr <- tiledb::tiledb_array(uri)
   tsch <- tiledb::schema(arr)
 
+  # XXX CHANGEME
```

Comment:
The massive line-counts on `test-SOMASparseNDArray.R`, `test-SOMADenseNDArray.R`, and `test-SOMADataFrame.R` are due mainly to using tiledb-r to introspect on compression filters etc. that we _written_ using tiledbsoma-r and `platform_config` but for which we lack a way to _read back and verify_ in tiledbsoma-r -- or tiledbsoma-py for that matter -- cc @nguyenv for thoughts ...